### PR TITLE
drivers: led: shell: print color as string

### PR DESCRIPTION
**Pull request by nixward**
Easier to use if human readable.

### CODE REVIEW

File: drivers/led/led_shell.c [hunk 1]
```diff
@@ -84,6 +85,27 @@ static int cmd_on(const struct shell *sh, size_t argc, char **argv)
 	return err;
 }
 
+static char *led_color(uint8_t color)
```
Comment (bbilas) [inline drivers/led/led_shell.c:88]:
```suggestion
static const char *led_color(uint8_t color)
```

File: drivers/led/led_shell.c [hunk 2]
```diff
@@ -84,6 +85,27 @@ static int cmd_on(const struct shell *sh, size_t argc, char **argv)
...
+	case LED_COLOR_ID_BLUE:
+		return "blue";
+	case LED_COLOR_ID_VIOLET:
+		return "violet";
+	case LED_COLOR_ID_YELLOW:
+		return "yellow";
+	case LED_COLOR_ID_IR:
+		return "IR";
+	}
+	return "unknown";
```
Comment (bbilas) [inline drivers/led/led_shell.c:106]: You can use 

<pre>
default:
    return "unknown"
</pre>

here.

File: drivers/led/led_shell.c [hunk 3]
```diff
@@ -84,6 +85,28 @@ static int cmd_on(const struct shell *sh, size_t argc, char **argv)
 	return err;
 }
 
+static const char *led_color(uint8_t color)
```
Comment (bbilas) [inline drivers/led/led_shell.c:88]: I would also rename that into e.g `led_color_to_str`.

Comment (nixward) [reply drivers/led/led_shell.c]: Done

**Review by bbilas — APPROVED** [review]

**Review by simonguinot — APPROVED** [review]

**Review by thedjnK — APPROVED** [review]

**Pull request merged**